package.json

```diff
@@ -3,18 +3,23 @@
   "private": true,
   "scripts": {
     "dev": "NODE_ENV=development next dev",
+    "win-dev": "next dev",
```

Comment:
We should set the env var here if possible, but *shrug*

Reply:
for me, running win-dev seemed to set NODE_ENV as well. The basepath is correctly "" in dev and "kata-dashboard-next" in prod (build/start).